openmeetings-web/src/main/java/org/apache/openmeetings/web/room/wb/raw-wb-area.js

```diff
@@ -248,7 +256,8 @@ var DrawWbArea = function() {
 			callback();
 		}
 		$('#wb-rename-menu').menu().find('.wb-rename').click(function() {
-			_getWbTab($(this).parent().data('wb-id')).find('a span').trigger('dblclick');
+			const textSpan = _getWbTab($(this).parent().data('wb-id')).find('.wb-nav-tab-text').first();
```

Comment:
this selector was wrong, prob just stopped working cause "a span" doesn't work. Anyhow, having a class as selector is more save in case this element gets moved inside DOM tree.

Reply:
And again using tags for selectors is not really best practise:
https://frontstuff.io/you-need-to-stop-targeting-tags-in-css

Reply:
great